var/spack/repos/builtin/packages/py-bmtk/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBmtk(PythonPackage):
```

Comment:
oh man, you have to do this for every new python package added to spack?

Reply:
Yes.